### DISCUSSION TITLE
Issue after deploying to IIS

### DISCUSSION THREAD

**Discussion by stevewasielewski** [microsoft/fluentui-blazor#2587] [General]
I have some issues after deploying to IIS:
![Screenshot 2024-08-27 153056](https://github.com/user-attachments/assets/6704651f-1a31-4014-a1db-9e1c72830945)

Here is the app.razor file:
<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <base href="/" />
    <link rel="stylesheet" href="app.css" />
    <link rel="stylesheet" href="LOGoals.styles.css" />
    <HeadOutlet />
</head>

<body>
    <Routes />
    <script src="_framework/blazor.web.js"></script>

</body>

</html>

Any Ideas on why this might be happening?

Thanks

**Comment by dvoituron** [comment]
Have you put your site in a subfolder? Then you need to configure `<base href="..." />`

**Reply by stevewasielewski** [reply]
Here is the app.razor file 
![Screenshot 2024-08-28 092600](https://github.com/user-attachments/assets/44212375-aa0b-4c37-a9ad-c304f338d278)
Here is the error stills:
![NewScreenshot 2024-08-28 092720](https://github.com/user-attachments/assets/bbdea5db-c6a9-480f-8af8-2843ea81cd35)

ideas?

**Reply by dvoituron** [reply]
You need to replace `...` by your sub-folder. See https://learn.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/?view=aspnetcore-8.0&tabs=visual-studio#configure-the-app-base-path
